### PR TITLE
registration working now

### PR DESCRIPTION
**Pull request by kolomoitsev**

### CODE REVIEW

**Comment by kolomoitsev** [comment]
need to add username field

File: src/components/register.component.js [hunk 1]
```diff
@@ -127,7 +130,7 @@ const Register = () => {
                     borderBottom: `1px solid red`,
```
Comment (DimonHamlet) [inline src/components/register.component.js:130]: First IF, where logic for bad pass is, must be `pass.length < 3 || pass !== repeatPass`

File: src/components/register.component.js [hunk 2]
```diff
@@ -290,6 +303,7 @@ const Register = () => {
 
                         </div>
 
+                        { registerError ? <p> error  </p> : null }
```
Comment (DimonHamlet) [inline src/components/register.component.js]: Need to handle error message, for example: there is already registered user
And same staff for login

**Review by DimonHamlet — CHANGES_REQUESTED** [review]
Need to be updated

File: src/components/register.component.js (hunk 1 above)
Comment (kolomoitsev) [reply src/components/register.component.js:130]: done .

File: src/components/register.component.js (hunk 2 above)
Comment (kolomoitsev) [reply src/components/register.component.js]: done.

Comment (kolomoitsev) [reply src/components/register.component.js]: everything fixed in this branch

**Review by DimonHamlet — APPROVED** [review]

**Pull request merged**